3p/recaptcha.js

```diff
@@ -77,10 +83,11 @@ window.initRecaptcha = function() {
    */
   let dataObject;
   try {
-    dataObject = parseJson(window.name);
+    dataObject = parseJson(win.name);
   } catch (e) {
-    dev().error(TAG + ' Could not parse the window name.');
-    return;
+    const errorMessage = ' Could not parse the window name.';
+    dev().error(TAG, '%s', errorMessage);
```

Comment:
dev error not needed if you plan to throw. uncaught exceptions are all reported.